examples/algorithms/using_existing/linspace.cpp

```diff
@@ -0,0 +1,49 @@
+#include <eve/module/core.hpp>
+#include <eve/algo.hpp>
+#include <iostream>
+#include <vector>
+#include <tuple>
+#include <eve/algo/concepts/relaxed.hpp>
+
+  template < eve::algo::relaxed_range R>
+  void print(R const & v)
+  {
+    std::cout << '{';
+    auto n = v.end() - v.begin();
+    auto cur = v.begin();
+    if(n)
+    {
+      for(auto i = 1;  i !=  n; ++i, ++cur) {std::cout << +eve::read(cur) << ", ";};
+      if (n) std::cout << +eve::read(cur) << "}\n";
+    }
+    else std::cout <<"}\n";
+  }
+
+// an accurate linspace with transform and lerp
+template < eve::algo::relaxed_range R, eve::scalar_value T,  eve::scalar_value U> R linspace(R& r, T a, U b )
+  requires( eve::floating_scalar_value<decltype(eve::read(r.begin()))>)
+{
+  EVE_ASSERT(eve::is_finite(a), "a is not finite");
+  EVE_ASSERT(eve::is_finite(b), "b is not finite");
+  using elt_t = decltype(eve::read(r.begin()));
+  std::size_t siz = r.end()-r.begin();
+  if (siz == 0u) return r;
+  else if (siz == 1u) {*r.begin() = elt_t(b); return r; }
+  else if (a == b)    { eve::algo::fill(r, elt_t(a)); return r; }
+  else
+  {
+    elt_t invnm1 = eve::rec((elt_t(eve::dec(siz))));
+    auto op = [a, b, invnm1](auto i){return eve::lerp(elt_t(a), elt_t(b), i*invnm1); };
+    eve::algo::transform_to[eve::algo::unroll<1>](eve::views::iota(elt_t{0}), r, op);
+    return r;
+  }
+}
+
+
+int main()
```

Comment:
Examples are tests.
See example

https://github.com/jfalcou/eve/blob/046b7e977ce67e9ddaaf9166759ea6e235795262/examples/algorithms/using_existing/inclusive_scan_par_unseq__using_eve_to_build_both_vectorized_and_parallel_algos.cpp#L225

Reply:
I know there is no need to copy this is just to show that the algo work on vector as well as on views.
now linspace is also a test. 
However all existing examples are not tests. (qick_start/constant.cpp or sanity_check are not)

Reply:
Joel created smth weird in the example folder. Please look at proper examples.

Reply:
> I know there is no need to copy this is just to show that the algo work on vector as well as on views.

No, you create a view + copy in a different algorithm. You are showing in a doc a bad pattern